agents/kubevirt/fence_kubevirt.py

```diff
@@ -127,8 +136,6 @@ def main():
 
     run_delay(options)
 
-    validate_options(['--namespace'], options)
```

Comment:
Seems like you can remove the function as well (I cant see it used anywhere else).